core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -58,7 +58,11 @@ private[ui] class ExecutorsPage(
     val diskUsed = storageStatusList.map(_.diskUsed).sum
     val execInfo = for (statusId <- 0 until storageStatusList.size) yield
       ExecutorsPage.getExecInfo(listener, statusId)
-    val execInfoSorted = execInfo.sortBy(_.id)
+    val execInfoSorted = execInfo.filter(x => {
```

Comment:
What is this doing? this is inefficient (creates regexes every time) and has strange syntax. Boolean or is not invoked as a method; extra parens.